js/main.js

```diff
@@ -95,3 +99,99 @@ var renderPins = function (adverts) {
 
 showMap();
 renderPins();
+
+// создаем DOM-элемент объявления (карточка объявления)
+
+var cardTemplate = document.querySelector('#card').content.querySelector('article');
+
+// создаем тип жилья
+var createTypeCard = function (card) {
+  switch (true) {
+    case card.offer.type === 'flat':
+      card.offer.type = 'Квартира';
+      break;
+
+    case card.offer.type === 'bungalo':
+      card.offer.type = 'Бунгало';
+      break;
+
+    case card.offer.type === 'house':
+      card.offer.type = 'Дом';
+      break;
+    case card.offer.type === 'palace':
+      card.offer.type = 'Дворец';
+      break;
+  }
+  return card.offer.type;
+};
+
+// создаем удобства в месте проживания
+var createFeature = function (container, feature) {
+  for (var i = 0; i < feature.length; i++) {
+    var featureItem = document.createElement('li');
+    featureItem.classList.add('popup__feature');
+    featureItem.classList.add('popup__feature--' + feature[i]);
+    container.querySelector('.popup__features').appendChild(featureItem);
+  }
+};
+
+// создаем фото жилья
+var createPhoto = function (container, photos) {
+  for (var i = 0; i < photos.length; i++) {
+    var photosItem = document.createElement('img');
+    photosItem.classList.add('popup__photo');
+    container.querySelector('.popup__photo').src = photos[i];
```

Comment:
здесь и далее по коду
```suggestion
    photosItem.src = photos[i];
```
container.querySelector('.popup__photo') всегда будет выбирать первый элемент, даже если фотографий больше чем одна, новый добавляются, но обновляется только первая. сейчас можно посмотреть в разметки img тегов несколько, а атрибуты только у первой картинки, потому что они перезаписываются

тоже лучше работать с фрагментом